dotCMS/src/main/java/com/dotmarketing/quartz/QuartzUtils.java

```diff
@@ -430,35 +439,31 @@ public static void removeTaskRuntimeValues(String jobName, String jobGroup) {
 	 * @throws ParseException 
 	 * @throws ClassNotFoundException 
 	 */
-	public static void scheduleTask(ScheduledTask job) throws SchedulerException, ParseException, ClassNotFoundException {
+	public static void scheduleTask(final ScheduledTask job) throws SchedulerException, ParseException, ClassNotFoundException {
 
-		Scheduler sched;
-		if (job.isSequentialScheduled())
-			sched = DotSchedulerFactory.getInstance().getSequentialScheduler();
-		else
-			sched = DotSchedulerFactory.getInstance().getScheduler();
+		final Scheduler scheduler = getScheduler();
 
 		JobDetail jobDetail;
 		Trigger trigger;
 		boolean isNew;
 
 		isNew = false;
 
-		String jobName = job.getJobName();
-		String jobGroup = job.getJobGroup();
-		String triggerName = job.getTriggerName() == null ? jobName + "_trigger" : job.getTriggerName();
-		String triggerGroup = job.getTriggerGroup() == null ? jobGroup : job.getTriggerGroup();
-		Date startDate = job.getStartDate();
-		Date endDate = job.getEndDate();
+		final String jobName = job.getJobName();
+		final String jobGroup = job.getJobGroup();
+		final String triggerName = job.getTriggerName() == null ? jobName + "_trigger" : job.getTriggerName();
+		final String triggerGroup = job.getTriggerGroup() == null ? jobGroup : job.getTriggerGroup();
+		final Date startDate = job.getStartDate();
+		final Date endDate = job.getEndDate();
 
-		if ((jobDetail = sched.getJobDetail(jobName, jobGroup)) == null) {
+		if ((jobDetail = scheduler.getJobDetail(jobName, jobGroup)) == null) {

```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6213816)